src/modules/edit.js

```diff
@@ -0,0 +1,16 @@
+export default class Edit {
+  constructor(e) {
+    this.event = e;
+    this.todos = JSON.parse(localStorage.getItem('todos')) || [];
+  }
+
+  editItem() {
+    const todo = this.todos.find((t) => t.index === +this.event.target.parentNode.id);
+    const edited = this.event.target.textContent.trim();
+
+    if (edited) {
+      todo.description = edited;
+    }
+    localStorage.setItem('todos', JSON.stringify(this.todos));
+  }
```

Comment:
- [ ] When I edit todo, it is not saved in local storage, kindly fix this issue and make sure after edit todo, it is saved in local storage.

Reply:
@omarsalem7 i already implemented that and its working fine on my side